src/utils.coffee

```diff
@@ -151,6 +151,7 @@ exports.getConfig = getConfig = (key) ->
 		return conf?.value
 
 exports.setConfig = (key, value) ->
+	value = null if _.isUndefined(value)
```

Comment:
This can be `exports.setConfig = (key, value = null) ->`, it'll set value to `null` if it's either `undefined` or `null`, which is equivalent